articles/aks/operator-best-practices-cluster-security.md

```diff
@@ -53,6 +53,10 @@ For more information about Azure AD integration, Kubernetes RBAC, and Azure RBAC
 >
 > Add a network policy in all user namespaces to block pod egress to the metadata endpoint.
 
+> [!Note]
+> For Network Policy Cluster needs to be created with attribute `--network-policy azure`
+> Example: `az aks create -g myResourceGroup -n myManagedCluster --enable-managed-identity --network-plugin azure --network-policy azure`
```

Comment:
Remove the word example and just retain the code block to complete the rewording / formatting of your added note. Thanks!

Reply:
completed